src/sonic-config-engine/minigraph.py

```diff
@@ -1458,6 +1458,29 @@ def select_mmu_profiles(profile, platform, hwsku):
                 base_file = os.path.join(path, file_item)
                 exec_cmd(["sudo", "cp", file_in_dir, base_file])
 
+def is_ipv4(address):
+    # address maybe is bytes type, so encode and decode to unicode
+    # set strict to False because address may set host bit, for example 192.168.0.1/24
+    return type(ipaddress.ip_network(address.encode().decode(), False)) is ipaddress.IPv4Network
+
+def update_forced_mgmt_route(mgmt_intf, mgmt_routes):
+    for mgmt_intf_key in mgmt_intf.keys():
+        forced_mgmt_routes = []
+
+        # get mgmt interface type
+        mgmt_intf_addr = mgmt_intf_key[1]
+        mgmt_is_ipv4 = is_ipv4(mgmt_intf_addr)
```

Comment:
>is_ipv4 [](http://example.com/codeflow?start=23&length=7)

If it throw due to invalid input data, is it good idea to log an error and continue the main flow?

Reply:
Fixed, write error message to stderr, because other exception handler in the file also write to stderr.